runtime/runtime/src/lib.rs

```diff
@@ -118,7 +118,7 @@ pub struct ApplyResult {
 }
 
 /// Stores indices for a persistent queue for delayed receipts that didn't fit into a block.
-#[derive(Default, BorshSerialize, BorshDeserialize)]
+#[derive(Default, BorshSerialize, BorshDeserialize, Serialize)]
```

Comment:
Where do you need to serialize this?

Reply:
Never. I only need to add this for `PeerMessage` and its "dependencies", but they turn to be so many, that I added `Serialize` to every struct that already had `BorshSerialize`.

Reply:
btw, already removed `Serialize` from here. Do you think I should revisit all added `Serialize`?